api/models/Notification.js

```diff
@@ -298,19 +298,30 @@ module.exports = bookshelf.Model.extend({
   },
 
   updateUserSocketRoom: function (userId) {
+    const { activity } = this.relations
     const actor = Object.assign({},
       this.actor().pick([ 'id', 'name' ]),
       { avatarUrl: this.actor().get('avatar_url') }
     )
+    const meta = activity.get('meta')
+    const action = Notification.priorityReason(meta.reasons)
     const comment = this.comment().pick([ 'id', 'text' ])
-    const community = this.relations.activity.relations.community.pick([ 'id', 'name', 'slug' ])
+    const community = activity.relations.community.pick([ 'id', 'name', 'slug' ])
+    const createdAt = activity.get('created_at').toString()
     const post = { id: this.post().id, title: this.post().get('name') }
```

Comment:
minor quibble -- if this line is only used in the final payload object, i'd rather just see it listed in-line there. but up to you